tools/riscv-openocd-ci/process_test_results.py

```diff
@@ -0,0 +1,172 @@
+
```

Comment:
I'm OK making changes to riscv-tests/debug to make this script simpler/

Reply:
That would be great. Either way, those improvements can be done incrementally in smaller steps. For a start, I am perfectly OK with the script as it is, later it can be simplified.